infrastructure/terraform/terraform.tfvars

```diff
@@ -0,0 +1,9 @@
+primary_location = "West Europe"
+secondary_location = "North Europe"
+
+organization_name = "agid"
+
+app_service_portal_post_login_url = "https://developer.cd.italia.it/developer"
+app_service_portal_post_logout_url = "https://developer.cd.italia.it/"
+adb2c_policy_name = "B2C_1_SignUpIn"
+apim_sku = "Developer"
```

Comment:
we can safely upgrade to 'Standard' (or Premium if we want vnet integration)

Reply:
it's not clear, why are these values here ? aren't they taken from the tfvars.json configuration file ?

Reply:
this file is going to replace tfvars.json

name of resources now happens inside each module by composing org name + environment, so we need few variables

Reply:
note that the variables in tfvars.json are shared and read by the nodejs tasks (which is the reason why it is a json).

Reply:
since that file is going to be edited by humans, it's best to use a human-friendly configuration syntax - anyway I'll work on the integration scripts after I'm done with terraform